windows-iotcore/develop-your-app/AppInstaller.md

```diff
@@ -9,9 +9,12 @@ keywords: windows iot, app installation, Windows Device Portal, devices
 ---
 
 # Install your app on an IoT Core device
-You can install your app by various methods and they are listed below.
+You can install your app using the one of the two methods that are listed below.
 
 ## Using Windows Device Portal
+
+For this method, you will need to ensure that you are connected to the internet. If you do not have access to the internet, you can also have a peer-to-peer ethernet connection between the device and a client machine that doesn't include a path to access the open internet.
+
```

Comment:
If the app is store signed and the connect is peer-to-peer, then the app will install but launching it will fail.   If the device does not have internet access to the store update server then the provisioning package method should be used.

Reply:
Thanks!